src/Integration/MefServices/IActiveSolutionBoundTracker.cs

```diff
@@ -18,5 +18,11 @@ public interface IActiveSolutionBoundTracker : IDisposable
         /// Returns whether the active solution is bound to a SonarQube project
         /// </summary>
         bool IsActiveSolutionBound { get; }
+
+        /// <summary>
+        /// Event to notify subscribers when the analyzers of a solution might have changed.
+        /// This occurs when a new solution is opened, or the SonarQube binding status of the solution changes.
+        /// </summary>
+        event EventHandler AnalyzersChanged;
```

Comment:
This event doesn't seemed to fit the interface (has nothing analyzer specific about it).
My current assumtion is that all the analyzer specific code should be in the VSIX, rather than in this class.
Also, in practice you are just raising the solution changed event anyway, so why this even is needed at all? Can you just use solutionTracker.ActiveSolutionChanged event instead of this one?